src/Components/LandingPage/LandingPage.js

```diff
@@ -19,12 +21,12 @@ export const LandingPage = () => {
 			try {
 				const response = await fetchRestaurantsData()
 				const restaurants = await response.json()
-				console.log(restaurants.data)
-				setRestaurantsData(restaurants.data)
+				history.push({pathname: '/selection', state:{restaurantsData: restaurants.data}})
```

Comment:
Ive been cautioned to avoid using History, lets see if we can find a way to do it without using history when we refactor. But for now I think its a good working solution.